core/README.md

```diff
@@ -1,4 +1,15 @@
-core
+# core/libdreamcast
 ===========
 
-All of the interesting bits are here
\ No newline at end of file
+Here lies the core of our codebase. Everything that's OS inspecific rests here.
+** Please check per directory README for more info **
+
+### Some rudimentary categories are:
+- hw				-- DC Hardware Components Implementation
+- nullDC.cpp		-- NullDC, thy mighty child (also referenced as "debugger")
+- emitter			-- Cookie machine 
+- khronos			-- Vulkan stuff
+- oslib				-- Codebase abstraction effort
+- cfg				-- Configuration backend structure
+- reios				-- (Our)Implementation of the DreamCast BIOS (Not functional)
+- deps				-- External C libraries (hackish, hand-written versions)
```

Comment:
Minor nit: the indentation is not the same on all the above.
Since this doesn't change anything regarding the display of the markdown it's fine by me (however I would like to have this changed).

Reply:
Done.